integration-tests/otlp/src/main/java/io/opentelemetry/integrationtest/OtlpExporterIntegrationTest.java

```diff
@@ -672,7 +666,6 @@ private static void testLogRecordExporter(LogRecordExporter logRecordExporter) {
           .setSeverityText("DEBUG")
           .setContext(Context.current())
           .emit();
-      eventLogger.builder("namespace.event-name").put("key", "value").emit();
```

Comment:
Generally in this PR, I translate any events to the equivalent log record call, rather than deleting. Here I delete because there's already a log record emitted / asserted which contains all the fields the event would